async-request-reply/README.md

```diff
@@ -157,15 +157,16 @@ resource serviceBusReceiverRoleAssignmentUser 'Microsoft.Authorization/roleAssig
 }
 
 // Assign Role to allow Read, write, and delete Azure Storage containers and blobs. 
-resource dataStorageBlobDataContributorRoleAssignmentUser 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
-  name: guid(resourceGroup().id, 'LocalUser', 'StorageBlobDataContributorRole')
+resource dataStorageBlobDataContributorRoleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
+  name: guid(resourceGroup().id, dataStorageAccount.id, 'StorageBlobDataContributorRole')
```

Comment:
Don't you still need 'LocalUser' in here?  This is a temporary role assignment for the user?  The others have 'LocalUser' as a component of this guid randomization.  Should they have the resource ID as well?

Reply:
I changed, I used the same name that I used to the function app, adding  'LocalUser' to make difference from the funtion app name.